src/app/core/learning-object-module/learning-object/learning-object.routes.ts

```diff
@@ -7,12 +7,15 @@ export const LEARNING_OBJECT_ROUTES = {
      * @param learningObjectId the id of the learning object
      * @returns void
      */
+    GET_CHILDREN(learningObjectID: string) {
+        return `${environment.apiURL}/learning-objects/${encodeURIComponent(learningObjectID)}/children`;
+    },
```

Comment:
There's a block comment that belongs to the UPDATE_LEARNING_OBJECT_STATUS route function below. Can you please rearange and also create a block comment for this route. Just a generic description like the other block comment